tests/Resources/Page/ODH/ODHDashboard/ODHModelServing.resource

```diff
@@ -0,0 +1,165 @@
+
+Verify Model Status
+    [Documentation]    Checks the reported model status in the Model Serving page in the dashboard.
+    ...    status can be "success" or any other string (which will expect failure status). this does NOT work
+    ...    for the status in the "Model server" section of a DSP.
+    [Arguments]    ${model_name}    ${expected_status}=success
+    IF    "${expected_status}"=="success"
+        Wait Until Keyword Succeeds    30s    5s    Page Should Contain Element    //div[.="${model_name} "]${MS_TABLE_STATUS}${MS_TABLE_STATUS_SUCCESS}
+        Page Should Not Contain Element    //div[.="${model_name} "]${MS_TABLE_STATUS}${MS_TABLE_STATUS_FAILURE}
+    ELSE
+        Wait Until Keyword Succeeds    30s    5s    Page Should Contain Element    //div[.="${model_name} "]${MS_TABLE_STATUS}${MS_TABLE_STATUS_FAILURE}
+        Page Should Not Contain Element    //div[.="${model_name} "]${MS_TABLE_STATUS}${MS_TABLE_STATUS_SUCCESS}
+    END
+
+Delete Model Via UI
+    [Documentation]    Deletes a deployed model from the Model Serving page in the dashboard.
+    [Arguments]    ${model_name}
+    Click Element    //div[.="${model_name} "]${MS_TABLE_DROPDOWN_TOGGLE}
+    Click Element    //div[.="${model_name} "]${MS_TABLE_DROPDOWN_TOGGLE}/..//button[.="Delete"]
+    Wait Until Page Contains Element    xpath://input[@id="delete-modal-input"]
+    Input Text    xpath://input[@id="delete-modal-input"]    ${model_name}
+    Click Button    Delete deployed model
+    Wait Until Page Does Not Contain Element    //div[.="${model_name} "]
+
+Open Model Edit Modal
+    [Documentation]    Opens the modal used to edit the details of an already deployed model.
+    [Arguments]    ${model_name}
+    Click Element    //div[.="${model_name} "]${MS_TABLE_DROPDOWN_TOGGLE}
+    Click Element    //div[.="${model_name} "]${MS_TABLE_DROPDOWN_TOGGLE}/..//button[.="Edit"]
+    Wait Until Page Contains    xpath://h1[.="Deploy model"]
+
+Get Model Route via UI
+    [Documentation]    Grabs the serving route (URL) of an already deployed model from the Model Serving page.
+    [Arguments]    ${model_name}
+    ${url}=    Get Element Attribute    //div[.="${model_name} "]${MS_TABLE_ENDPOINT_INPUT}    value
+    [Return]    ${url}
+
+Open ${section} Options Menu
+    [Documentation]    Opens the "Options menu" dropdown for different sections
+    ...    Valid sections are: "Name", "Model framework", "Project"
+    Click Element    xpath://span[.="${section}"]/../../..//button[@aria-label="Options menu"]
+
+    # //div[.="test-model-no-token "]/../../td[@data-label="Project"] (.= project name)
+    # //div[.="test-model-no-token "]/../../td[@data-label="Inference endpoint"]//div[@class="pf-c-clipboard-copy__group"]/input (value=url)
+    # //div[.="test-model-no-token "]/../../td[@data-label="Status"]//span[contains(@class,"pf-c-icon__content")] (class=status?) (pf-m-danger > failed)  (pf-m-danger) (pf-m-success)
```

Comment:
do you need these lines under comment?

Reply:
good catch, I don't, it was just a bunch of xpaths that I could need